BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -48,17 +48,14 @@ public struct ElementContent {
         switch layoutMode {
         case .standard:
             return storage.measure(in: constraint, environment: environment, cache: cache)
-        case .singlePass:
+        case .singlePass(let options):
             return storage.sizeThatFits(
                 proposal: constraint,
                 context: .init(
                     // TODO: Hoist upward?
                     cache: SPCacheNode(
                         path: "m",
-                        options: SPCacheOptions(
-                            hintRangeBoundaries: true,
-                            searchUnconstrainedKeys: false
```

Comment:
Are we sure we want to use a valid of `false` for `searchCnconstrainedKeys` here? Do we have a TODO list item to dig deeper into the performance trade-off here?

Reply:
Hm, I thought you had convinced me previously that it didn't matter during measurement, but I could be misremembering. Anyhow, I added an item to come back to this.